FirstDeliverable/CSV/csv.go

```diff
@@ -0,0 +1,67 @@
+package csv
+
+import (
+	"encoding/csv"
+	"encoding/json"
+	"fmt"
+	"io"
+	"log"
+	"os"
+)
+
+type Item struct {
+	ItemID   string `json:"item_id"`
+	ItemName string `json:"item_name"`
+	ItemType string `json:"item_type"`
+}
+
+func ReadCSV() ([][]string, error) {
+	// Open the file
+	csvfile, err := os.Open("../FirstDeliverable/CSV/result.csv")
```

Comment:
As a nice to have:
You should move this to a config file, so the file will be opened when the application runs and not in every request.